### PR TITLE
Style header navigation to resemble game ui

### PR DESCRIPTION
**Pull request by leichteckig**
My first try to make the header look like in-game. WIP!~

### CODE REVIEW

**Comment by leichteckig** [comment]
Reference
image

I want to stick to the reference as far as possible, including these icons for mobile purposes.

**Comment by leichteckig** [comment]
Successfully rebased. :)

File: src/components/UiButton.vue [hunk 1]
```diff
@@ -1,15 +1,30 @@
 <template>
     <router-link class="ui-button" :class="classes" v-if="routerLink" :to="routerLink">
-        <span v-if="hasIconSlot" class="ui-button__icon">
-            <slot name="icon"></slot>
-        </span>
-        <slot></slot>
+        <div :class="'ui-blood--' + this.direction">
+            <div :class="'ui-button__limiter--' + this.direction">
```
Comment (3stadt) [inline src/components/UiButton.vue]: `this` not needed here

File: src/components/UiButton.vue [hunk 2]
```diff
@@ -24,13 +39,21 @@ export default {
...
+    direction: {
+      type: String,
+      default: 'left',
+      required: true
     }
   },
 
   computed: {
     classes () {
+      console.log(this.direction)
```
Comment (3stadt) [inline src/components/UiButton.vue]: debug code

Comment (leichteckig) [reply src/components/UiButton.vue]: :heavy_check_mark:

File: src/components/UiButton.vue (hunk 1 above)
Comment (leichteckig) [reply src/components/UiButton.vue]: :heavy_check_mark:

**Pull request merged**